dashboard/templates/dashboard/resubmit_pset_form.html

```diff
@@ -29,4 +29,16 @@ <h2>{{ pset.unit }}</h2>
   </p>
   <hr />
   {% include "generic_form.html" %}
+  <br />
+  <div class="alert alert-info">
+    If you have problems to suggest for this unit,
+    please use the <a href="{% url "suggest-new" %}">problem suggestion form</a>.
+  </div>
 {% endblock layout-content %}
+{% block scripts %}
+  <script type="text/javascript">
+      $(() => {
+          $("select").chosen();
+      });
+  </script>
+{% endblock scripts %}
```

Comment:
there should be a newline at the end of the file

Reply:
not literally an empty line --- i just mean there shouldn't be this minus sign (no-eol)
<img width="270" height="128" alt="image" src="https://github.com/user-attachments/assets/9d13f039-056b-48a5-af36-2947785c1de7" />